src/pages/Auth/Auth.css

```diff
@@ -94,4 +107,37 @@
       font-size: 1.49rem;
       margin-left: 12rem;
   }
- 
\ No newline at end of file
+ .visibility{
+   display: flex;
+   align-items: center;
+   border: 1px solid var(--gray);
+   width: 80%;
+   margin: 5px 2rem !important;
+  }
+  .visibility-icon{
+    padding-right: 2rem;
+    color: var(--primary-color);
+    font-size: 2rem;
+  }
+  .visibility-fields{
+    display: block;
+    width: 100% !important;
+    border: none !important;
```

Comment:
you used !important a lot, please use stylings using the order it might reduce the use of !important

Reply:
Sure will modify it